java.en.md

```diff
@@ -0,0 +1,179 @@
+# Table of Contents
+
+- [Style for Java](#style-for-java)
+ - [Imports](#imports)
+ - [Indentation](#indentation)
+ - [Whitespaces](#whitespaces)
+ - [Where to break](#where-to-break)
+ - [Braces](#braces)
+ - [Line columns](#line-columns)
+ - [Acronyms](#acronyms)
+ - [TODO](#todo)
+ - [Arrays](#arrays)
+- [Style for Android](#style-for-android)
+ - [Field names](#field-names)
+ - [Resources](#resources)
+
+# Style for Java
+
+## Introduction
+
+This document defines the conventions for writing Java code at COOKPAD Inc.
+These regulations are formed with respect to readability and maintainability, such that developers familiar with Java can easily read the code.
+To ensure readability and consistency within the code, the guide presents a number of rules to follow. Some of these rules must be followed, while others are simply a recommendation for the developer.
+
+## Imports
+
+- **[MUST]** Remove unused imports.
+- **[MUST]** Wildcard imports, static or otherwise, are not used.
```

Comment:
We already use static imports mainly for testing with JUnit4.

Should we make more concrete rules?